pyvista/core/cell.py

```diff
@@ -356,13 +356,34 @@ def bounds(self) -> Tuple[float, float, float, float, float, float]:
 
         Examples
         --------
-        >>> import pyvista
-        >>> mesh = pyvista.Sphere()
+        >>> import pyvista as pv
+        >>> mesh = pv.Sphere()
         >>> mesh.get_cell(0).bounds
         (-0.05405950918793678, -5.551115123125783e-17, 0.0, 0.011239604093134403, -0.5, -0.49706897139549255)
+
         """
         return self.GetBounds()
 
+    @property
+    def center(self) -> Tuple[float, float, float]:
+        """Get the center of the cell.
+
+        Examples
+        --------
+        >>> import pyvista as pv
+        >>> mesh = pv.Sphere()
+        >>> mesh.get_cell(0).center
+        (-0.0356, 0.00375, -0.498)
+
+        """
+        para_center = [0.0, 0.0, 0.0]
+        sub_id = self.GetParametricCenter(para_center)
+        sub_id = _vtk.mutable(0)
+        center = [0.0, 0.0, 0.0]
+        weights = [0.0] * self.n_points
+        self.EvaluateLocation(sub_id, para_center, center, weights)
+        return cast(Tuple[float, float, float], tuple(center))
```

Comment:
`DataSet.center` returns a list, however this is return is immutable.  I think I remember there being some discussion about this, but forget where it landed.

Reply:
Last thing I could find was https://github.com/pyvista/pyvista/pull/3747.

Reply:
Thanks, I think this usage is consistent with that.